pjlib/src/pj/ioqueue_winnt.c

```diff
@@ -586,6 +611,12 @@ PJ_DEF(pj_status_t) pj_ioqueue_register_sock( pj_pool_t *pool,
         return PJ_RETURN_OS_ERROR(GetLastError());
     }
 
+    /* Group lock */
+    rec->grp_lock = grp_lock;
+    if (rec->grp_lock) {
+        pj_grp_lock_add_ref_dbg(rec->grp_lock, "ioqueue", 0);
```

Comment:
- there should be the `pj_grp_lock_dec_ref()` pair?
- as in other ioqueue, the group lock `add/dec_ref()` is invoked surrounding event callbacks to prevent premature destroy/unregistration by other thread while the app is inside the callback?

Reply:
Ah, right. Integrating group lock will be a much more complicated task.

Just a bit of background, the group lock introduction was done in #1616 dated Feb 2013 for release 2.1 and was integrated into ioq select and even (the then-experimental) epoll backend. Ioq-winnt (IOCP) was notably missing and not sure whether it was simply overlooked or if there were some implementation challenges during the integration. But the obvious thing is that IOCP hasn't been working for more than a decade and there didn't seem to be much interest (?) either.

So for now I suggest the PR is just to fix the build failure. If more users start using this backend again, we may consider revisiting the group lock integration.